profile.js

```diff
@@ -35,4 +35,64 @@ document.getElementById('menu').addEventListener('click', () => {
   document.getElementById('link1').addEventListener('click', hideMobMenu);
   document.getElementById('link2').addEventListener('click', hideMobMenu);
   document.getElementById('link3').addEventListener('click', hideMobMenu);
-});
\ No newline at end of file
+});
+// || =========== Pop-Up Modal ============ ||
+const arrProjects = [];
+function addProjects() {
+  // Stores Project Infos in arrProject
+  const colTitles = document.getElementsByClassName('project-titles');
+  const colClient = document.getElementsByClassName('client');
+  const colRole = document.getElementsByClassName('type');
+  const colYear = document.getElementsByClassName('year');
+  for (let i = 0; i < colTitles.length; i += 1) {
+    // | creating a new project object an pushing it to arrProject
+    const objProject = {};
+    objProject.name = colTitles[i].textContent;
+    objProject.client = colClient[i].textContent;
+    objProject.role = colRole[i].textContent;
+    objProject.year = colYear[i].textContent;
+    objProject.tags = ['html', 'css', 'javascript', 'github', 'ruby', 'bootstrap'];
+    objProject.snapshot = `images/SnapshootD${i + 1}.jpg`;
+    arrProjects.push(objProject);
+  }
+}
+
+function getProjectDetails(proID) {
+  // Retrieving  Project Info
+  return arrProjects[proID];
+}
+
+function showProjectDetails(proID) {
+  // showing  Project Info
```

Comment:
- [ ] For mobile screens please fix the model buttons' width and make sure it matches the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/169249876-eccfc548-2d09-42f3-9be0-6af692dd41f5.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/169249920-d4af0878-3503-494e-b525-a8566c200588.png)

Reply:
Made All the Changes as suggested by you.